inventory/inventory.go

```diff
@@ -0,0 +1,72 @@
+package inventory
+
+import (
+	"github.com/opencontrol/compliance-masonry/models"
+	"github.com/opencontrol/compliance-masonry/tools/certifications"
+)
+
+// Inventory maintains the inventory of all the controls within a given workspace.
+type Inventory struct {
+	*models.OpenControl
+	masterControlList       map[string]models.Control
+	actualSatisfiedControls map[string]models.Satisfies
+	MissingControlList      map[string]models.Control
+}
+
+// standardAndControlString makes a string from the standard and the control.
+// This is helpful for functions that want to create unique keys consistently.
+func standardAndControlString(standard string, control string) string {
```

Comment:
both model.Control and model.Satisfies pass in their respective standard and and control to this function.
A couple of alternatives: 
- We need to create two `standardAndControlString` functions `standardAndControlString(model.Control)` and `standardAndControlString(model.Satisfies)`.
- But I think at that point we would make those functions part of the struct instead. e.g. `func (c Control) String() string` and `func (s Satisfies) String() string`
- Or leave as is.

I think I will do the second one. For long term maintainability and style